schemas/JSON/manifests/v1.1.0/manifest.singleton.1.1.0.json

```diff
@@ -165,20 +165,57 @@
         }
       }
     },
+    "InstallerReturnCode": {
+      "type": "integer",
+      "not": {
+        "enum": [ 0 ]
+      },
+      "minimum": -2147483648,
+      "maximum": 4294967295,
+      "description": "An exit code that can be returned by the installer after execution"
+    },
     "InstallerSuccessCodes": {
       "type": [ "array", "null" ],
       "items": {
-        "type": "integer",
-        "not": {
-          "enum": [ 0 ]
-        },
-        "minimum": -2147483648,
-        "maximum": 4294967295
+        "$ref": "#/definitions/InstallerReturnCode"
       },
       "maxItems": 16,
       "uniqueItems": true,
       "description": "List of additional non-zero installer success exit codes other than known default values by winget"
     },
+    "ExpectedReturnCodes": {
+      "type": [ "array", "null" ],
+      "items": {
+        "type": "object",
+        "properties": {
+          "InstallerReturnCode": {
+            "$ref": "#/definitions/InstallerReturnCode"
+          },
+          "ReturnResponse": {
+            "type": [ "string", "null" ],
+            "enum": [
+              "PackageInUse",
+              "InstallInProgress",
+              "FileInUse",
+              "MissingDependency",
+              "DiskFull",
+              "InsufficientMemory",
+              "NoNetwork",
+              "ContactSupport",
+              "RebootRequiredToFinish",
+              "RebootRequiredForInstall",
+              "RebootInitiated",
+              "CancelledByUser",
+              "AlreadyInstalled",
+              "Downgrade",
+              "BlockedByPolicy"
+            ]
+          }
+        }
+      },
+      "maxItems": 16,

```

Comment:
>16, [](http://example.com/codeflow?start=18&length=3)

Since we allow many to one mapping now, shall we increase the maxItems? I'd think 128

Also, please make the definition in this file and the other file consistent, the installer schema did not have maxItems